llms/mlx_lm/models/gpt_bigcode.py

```diff
@@ -0,0 +1,211 @@
+from dataclasses import dataclass
+from typing import Dict, Optional, Tuple, Union
+
+import mlx.core as mx
+import mlx.nn as nn
+import numpy as np
+
+from .base import BaseModelArgs
+
+
+@dataclass
+class ModelArgs(BaseModelArgs):
+    model_type: str
+    n_embd: int
+    n_layer: int
+    n_inner: int
+    n_head: int
+    n_positions: int
+    layer_norm_epsilon: float
+    vocab_size: int
+    num_key_value_heads: int = None
+    attn_pdrop: float = 0.1
+    embd_pdrop: float = 0.1
+    resid_pdrop: float = 0.1
```

Comment:
Remove these unused arguments.

Reply:
They are being used on the attention, embedding and mlp blocks :)

Reply:
Right let's remove those Dropout layers as well.. we don't add dropout layers to model files here since we don't use them for trianing.

Reply:
Got it!

Done ✅